install/deb/nginx/nginx.conf

```diff
@@ -71,48 +72,42 @@ http {
 	gzip_vary                       on;
 	gzip_comp_level                 6;
 	gzip_min_length                 1024;
-	gzip_buffers                    16 8k;
+	gzip_buffers                    128 4k;
 	gzip_http_version               1.1;
-	gzip_types                      text/plain text/css text/javascript text/js text/xml
-	  application/json application/javascript application/x-javascript application/xml
-	  application/xml+rss application/x-font-ttf image/svg+xml font/opentype;
+	gzip_types                      text/css text/javascript text/js text/plain text/richtext text/shtml text/x-component text/x-java-source text/x-markdown text/x-script text/xml image/bmp image/svg+xml image/vnd.microsoft.icon image/x-icon font/otf font/ttf font/x-woff multipart/bag multipart/mixed application/eot application/font application/font-sfnt application/font-woff application/javascript application/javascript-binast application/json application/ld+json application/manifest+json application/opentype application/otf application/rss+xml application/ttf application/truetype application/vnd.api+json application/vnd.ms-fontobject application/wasm application/xhtml+xml application/xml application/xml+rss application/x-httpd-cgi application/x-javascript application/x-opentype application/x-otf application/x-perl application/x-protobuf application/x-ttf;
 	gzip_proxied                    any;
-	gzip_disable                    "MSIE [1-6]\.";
-    
-	# Cloudflare ips
+	gzip_disable                    "msie6";
```

Comment:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=867024

Maybe it is time to just remove it... 

Microsoft IE 6 has been dead for